plugins/python_3_script/icon_python_3_script/connection/schema.py

```diff
@@ -54,38 +54,38 @@ class ConnectionSchema(insightconnect_plugin_runtime.Input):
       "type": "object",
       "title": "Credential: Secret Key",
       "description": "A shared secret key",
+      "required": [
+        "secretKey"
+      ],
```

Comment:
this needs to be moved to line 67 as the field `secretKey` is a required property  of the nested object and the top level 

eg it needs to be 
{"credential_secret_key":  {"secretKey": "required"}}

whereas the current code suggests
{"secretKey" : "required}